src/logger/DebugLogger.ts

```diff
@@ -0,0 +1,103 @@
+import * as debug from "debug";
```

Comment:
we need to dynamically load any third-party package using `require` from `PlatformTools` class. Its requirement for typeorm to be cross-platform

Reply:
```ts
import * as path from "path";

/**
 * Platform-specific tools.
 */
export class PlatformTools {
    ...
		
    /**
     * Debug module needed by DebugLogger
     */
    static debug = debug;
}
```
like that? what do you prefer?

Reply:
no, just see how many other libraries like mkdirp, redis, mysql, etc. are loaded using PlatformTools, its dynamic loading using `require` function

Reply:
ahh you meant `PlatformTools.load`